.github/labeler.yml

```diff
@@ -2,20 +2,33 @@
 Multicast:
   - changed-files:
     - any-glob-to-any-file:
-      - setup.py
+      - 'setup.*'
       - multicast/*.py
       - LICENSE
       - LICENSE.md
+      - pyproject.toml
 
 # Add 'Lint' label to any lint file changes
 Linter:
   - changed-files:
     - any-glob-to-any-file:
-      - .hound.yml
       - .yamllint.conf
       - .flake8.ini
       - .deepsource.toml
+      - tests/check_s*
+      - tests/check_c*
+      - tests/check_pip
+
+# Add 'bash lang' label to any bash file changes
+Bash Lang:
+  - changed-files:
+    - any-glob-to-any-file:
+      - tests/*.bash
       - tests/check_*
+      - .github/*.bash
+      - .github/*.sh
+      - .circleci/*.bash
+      - '*.bash'
```

Comment:
LGTM! Better configuration if some bash scripts are not linters (ie like non-linter testing)